test/system/groups/samples_test.rb

```diff
@@ -122,21 +122,21 @@ def retrieve_puids
       # they are ordered as expected against one another.
       assert_selector 'table tbody tr', count: 20
 
-      click_on I18n.t('groups.samples.table.puid')
+      click_on 'Sample ID'
```

Comment:
I think these should be internationalized

Reply:
I attempted to do this, but was unable to as the translations are stored with the component and not accessible during testing.

Reply:
I noticed that with another PR as well. We will probably need to figure something out eventually